code/modules/antagonists/heretic/knowledge/moon_lore.dm

```diff
@@ -0,0 +1,266 @@
+/**
+ * # The path of Moon.
+ *
+ * Goes as follows:
+ *
+ * Moonlight Troupe
+ * Grasp of Lunacy
+ * Smile of the moon
+ * > Sidepaths:
+ *   Mind Gate
+ *   Ashen Eyes
+ *
+ * Mark of Moon
+ * Ritual of Knowledge
+ * Lunar Parade
+ * Moonlight Amulette
+ * > Sidepaths:
+ *   Curse of Paralasys
+ *   Unfathomable Curio
+ * 	 Unsealed Arts
+ *
+ * Moonlight blade
+ * Ringleaders Rise
+ * > Sidepaths:
+ *   Ashen Ritual
+ *
+ * Last Act
+ */
+/datum/heretic_knowledge/limited_amount/starting/base_moon
+	name = "Moonlight Troupe"
+	desc = "Opens up the Path of Moon to you. \
+		Allows you to transmute 2 sheets of iron and a knife into an Lunar Blade. \
+		You can only create two at a time."
+	gain_text = "Under the light of the moon the laughter echoes."
+	next_knowledge = list(/datum/heretic_knowledge/moon_grasp)
+	required_atoms = list(
+		/obj/item/knife = 1,
+		/obj/item/stack/sheet/iron = 2,
+	)
+	result_atoms = list(/obj/item/melee/sickly_blade/moon)
+	route = PATH_MOON
+
+/datum/heretic_knowledge/base_moon/on_gain(mob/user, datum/antagonist/heretic/our_heretic)
+	add_traits(TRAIT_EMPATH, our_heretic)
```

Comment:
Not how add traits works, it's scoped on the datum you want to have the traits. Also don't pass a datum as a source. 
`ADD_TRAIT(user, TRAIT_EMPATH, "heretic")`